index.js

```diff
@@ -57,10 +57,11 @@ const inputs = [
 readSequential(inputs)
     .then(options => {
         return getCertificate(options).then(certs => {
+            if(!certs) return Promise.reject("Could not get certs. Check previous errors");
```

Comment:
Have you found a case where `getCertificate` returns falsy but the promise is resolved? This error shouldn't be handled here (defeats the purpose of returning a promise).

Also, please do not use 1-line blocks (always braces) and use single quotes `''`.